backend/tests/conftest.py

```diff
@@ -420,3 +420,90 @@ async def titles_with_metadata(language_fra, language_eng):
         for metadata in await title.metadata.all():
             await metadata.delete()
         await title.delete()
+
+
+@pytest.fixture(scope="function")
+@pytest.mark.asyncio
+async def titles_with_metadata_books(language_fra, book_dict):
+    """Title with a combination of books with and without Flavours"""
+
+    title_en = await Title.objects.create(ident="wikipedia_en_maxi")
```

Comment:
Following the [Convention](https://github.com/openzim/overview/wiki/Naming-Convention), can you use `wikipedia_en_all_maxi` ?